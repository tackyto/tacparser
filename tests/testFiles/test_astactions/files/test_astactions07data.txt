"Test01" {
    "FamilyName01" {
        {name = "Aaaa Aaaa", age = 41, sex = Man, phone = 0123-4567-8901 }
         {name = "Aaaa Bbbb", age = 42, sex = Female, phone = 012-34-8902 }
    }





    "FamilyName02" {
          {name = "Bbbb Aaaa", age = 45, sex = Man, phone = 0123-4567-8901 }
           {name = "Bbbb Bbbb", age = 44, sex = Female }
    }
 
 




    "FamilyName03" {
            {name = "Cccc Aaaa", age = 45, sex = Man, phone = 0123-4567-8903}
             {name = "Cccc Bbbb", age = 44, sex = Female }
    }
 
 
 
 
 
 
    "FamilyName04" {
              {name = "Aaaa Aaaa", age = 81, sex = M, phone = 0123-4567-8904 }
    }
 
 
 
 
 
 
    "FamilyName05" {
               {name = "Dddd Aaaa", age = 43, sex = Man, phone = 0123-4567-8905-1111 }
    }
}
