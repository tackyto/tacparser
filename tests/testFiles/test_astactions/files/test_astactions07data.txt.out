ASTActionsTest : (1, 0 - 43, 2 : 0 - 773) : {}
    Society : (1, 0 - 43, 2 : 0 - 773) : {}
        SocietyName : (1, 0 - 1, 9 : 0 - 9) : {}
            Literal : (1, 0 - 1, 9 : 0 - 9) : {}
                @Tarminal : (1, 1 - 1, 7 : 1 - 7) "Test01"
        Family : (2, 4 - 11, 4 : 15 - 198) : {}
            FamilyName : (2, 4 - 2, 19 : 15 - 30) : {}
                Literal : (2, 4 - 2, 19 : 15 - 30) : {}
                    @Tarminal : (2, 5 - 2, 17 : 16 - 28) "FamilyName01"
            Person : (3, 9 - 3, 73 : 41 - 105) : {endline: @EL < 4, endvolumn: @EC <= 77}
                Name : (3, 9 - 3, 29 : 41 - 61) : {}
                    Literal : (3, 16 - 3, 27 : 48 - 59) : {}
                        @Tarminal : (3, 17 - 3, 26 : 49 - 58) "Aaaa Aaaa"
                Age : (3, 29 - 3, 39 : 61 - 71) : {}
                    Number : (3, 35 - 3, 37 : 67 - 69) : {}
                        @Tarminal : (3, 35 - 3, 37 : 67 - 69) "41"
                Sex : (3, 39 - 3, 50 : 71 - 82) : {}
                    MorF : (3, 45 - 3, 48 : 77 - 80) : {}
                        Male : (3, 45 - 3, 48 : 77 - 80) : {}
                Phone : (3, 50 - 3, 73 : 82 - 105) : {}
                    NumberHyphen : (3, 58 - 3, 73 : 90 - 105) : {}
                        @Tarminal : (3, 58 - 3, 63 : 90 - 95) "0123-"
                        @Tarminal : (3, 63 - 3, 68 : 95 - 100) "4567-"
                        @Tarminal : (3, 68 - 3, 72 : 100 - 104) "8901"
            Person : (4, 10 - 4, 74 : 117 - 181) : {endvolumn: @EC <= 77}
                Name : (4, 10 - 4, 30 : 117 - 137) : {}
                    Literal : (4, 17 - 4, 28 : 124 - 135) : {}
                        @Tarminal : (4, 18 - 4, 27 : 125 - 134) "Aaaa Bbbb"
                Age : (4, 30 - 4, 40 : 137 - 147) : {}
                    Number : (4, 36 - 4, 38 : 143 - 145) : {}
                        @Tarminal : (4, 36 - 4, 38 : 143 - 145) "42"
                Sex : (4, 40 - 4, 54 : 147 - 161) : {}
                    MorF : (4, 46 - 4, 52 : 153 - 159) : {}
                        Female : (4, 46 - 4, 52 : 153 - 159) : {}
                Phone : (4, 54 - 4, 74 : 161 - 181) : {}
                    NumberHyphen : (4, 62 - 4, 74 : 169 - 181) : {}
                        @Tarminal : (4, 62 - 4, 66 : 169 - 173) "012-"
                        @Tarminal : (4, 66 - 4, 69 : 173 - 176) "34-"
                        @Tarminal : (4, 69 - 4, 73 : 176 - 180) "8902"
        Family : (11, 4 - 21, 4 : 198 - 367) : {}
            FamilyName : (11, 4 - 11, 19 : 198 - 213) : {}
                Literal : (11, 4 - 11, 19 : 198 - 213) : {}
                    @Tarminal : (11, 5 - 11, 17 : 199 - 211) "FamilyName02"
            Person : (12, 11 - 12, 75 : 226 - 290) : {endvolumn: @EC <= 77, startcolumn: @C >= 11, startlineeq: @L == 12}
                Name : (12, 11 - 12, 31 : 226 - 246) : {}
                    Literal : (12, 18 - 12, 29 : 233 - 244) : {}
                        @Tarminal : (12, 19 - 12, 28 : 234 - 243) "Bbbb Aaaa"
                Age : (12, 31 - 12, 41 : 246 - 256) : {}
                    Number : (12, 37 - 12, 39 : 252 - 254) : {}
                        @Tarminal : (12, 37 - 12, 39 : 252 - 254) "45"
                Sex : (12, 41 - 12, 52 : 256 - 267) : {}
                    MorF : (12, 47 - 12, 50 : 262 - 265) : {}
                        Male : (12, 47 - 12, 50 : 262 - 265) : {}
                Phone : (12, 52 - 12, 75 : 267 - 290) : {}
                    NumberHyphen : (12, 60 - 12, 75 : 275 - 290) : {}
                        @Tarminal : (12, 60 - 12, 65 : 275 - 280) "0123-"
                        @Tarminal : (12, 65 - 12, 70 : 280 - 285) "4567-"
                        @Tarminal : (12, 70 - 12, 74 : 285 - 289) "8901"
            Person : (13, 12 - 13, 55 : 304 - 347) : {endvolumn: @EC <= 77, startcolumn: @C >= 11}
                Name : (13, 12 - 13, 32 : 304 - 324) : {}
                    Literal : (13, 19 - 13, 30 : 311 - 322) : {}
                        @Tarminal : (13, 20 - 13, 29 : 312 - 321) "Bbbb Bbbb"
                Age : (13, 32 - 13, 42 : 324 - 334) : {}
                    Number : (13, 38 - 13, 40 : 330 - 332) : {}
                        @Tarminal : (13, 38 - 13, 40 : 330 - 332) "44"
                Sex : (13, 42 - 13, 55 : 334 - 347) : {}
                    MorF : (13, 48 - 13, 55 : 340 - 347) : {}
                        Female : (13, 48 - 13, 55 : 340 - 347) : {}
        Family : (21, 4 - 31, 4 : 367 - 543) : {}
            FamilyName : (21, 4 - 21, 19 : 367 - 382) : {}
                Literal : (21, 4 - 21, 19 : 367 - 382) : {}
                    @Tarminal : (21, 5 - 21, 17 : 368 - 380) "FamilyName03"
            Person : (22, 13 - 22, 76 : 397 - 460) : {endvolumn: @EC <= 77, startcolumn: @C >= 11}
                Name : (22, 13 - 22, 33 : 397 - 417) : {}
                    Literal : (22, 20 - 22, 31 : 404 - 415) : {}
                        @Tarminal : (22, 21 - 22, 30 : 405 - 414) "Cccc Aaaa"
                Age : (22, 33 - 22, 43 : 417 - 427) : {}
                    Number : (22, 39 - 22, 41 : 423 - 425) : {}
                        @Tarminal : (22, 39 - 22, 41 : 423 - 425) "45"
                Sex : (22, 43 - 22, 54 : 427 - 438) : {}
                    MorF : (22, 49 - 22, 52 : 433 - 436) : {}
                        Male : (22, 49 - 22, 52 : 433 - 436) : {}
                Phone : (22, 54 - 22, 76 : 438 - 460) : {}
                    NumberHyphen : (22, 62 - 22, 76 : 446 - 460) : {}
                        @Tarminal : (22, 62 - 22, 67 : 446 - 451) "0123-"
                        @Tarminal : (22, 67 - 22, 72 : 451 - 456) "4567-"
                        @Tarminal : (22, 72 - 22, 76 : 456 - 460) "8903"
            Person : (23, 14 - 23, 57 : 476 - 519) : {endvolumn: @EC <= 77, startcolumn: @C >= 11}
                Name : (23, 14 - 23, 34 : 476 - 496) : {}
                    Literal : (23, 21 - 23, 32 : 483 - 494) : {}
                        @Tarminal : (23, 22 - 23, 31 : 484 - 493) "Cccc Bbbb"
                Age : (23, 34 - 23, 44 : 496 - 506) : {}
                    Number : (23, 40 - 23, 42 : 502 - 504) : {}
                        @Tarminal : (23, 40 - 23, 42 : 502 - 504) "44"
                Sex : (23, 44 - 23, 57 : 506 - 519) : {}
                    MorF : (23, 50 - 23, 57 : 512 - 519) : {}
                        Female : (23, 50 - 23, 57 : 512 - 519) : {}
        Family : (31, 4 - 40, 4 : 543 - 661) : {}
            FamilyName : (31, 4 - 31, 19 : 543 - 558) : {}
                Literal : (31, 4 - 31, 19 : 543 - 558) : {}
                    @Tarminal : (31, 5 - 31, 17 : 544 - 556) "FamilyName04"
            Person : (32, 15 - 32, 77 : 575 - 637) : {endvolumn: @EC <= 77, startcolumn: @C >= 11}
                Name : (32, 15 - 32, 35 : 575 - 595) : {}
                    Literal : (32, 22 - 32, 33 : 582 - 593) : {}
                        @Tarminal : (32, 23 - 32, 32 : 583 - 592) "Aaaa Aaaa"
                Age : (32, 35 - 32, 45 : 595 - 605) : {}
                    Number : (32, 41 - 32, 43 : 601 - 603) : {}
                        @Tarminal : (32, 41 - 32, 43 : 601 - 603) "81"
                Sex : (32, 45 - 32, 54 : 605 - 614) : {}
                    MorF : (32, 51 - 32, 52 : 611 - 612) : {}
                        Male : (32, 51 - 32, 52 : 611 - 612) : {}
                Phone : (32, 54 - 32, 77 : 614 - 637) : {}
                    NumberHyphen : (32, 62 - 32, 77 : 622 - 637) : {}
                        @Tarminal : (32, 62 - 32, 67 : 622 - 627) "0123-"
                        @Tarminal : (32, 67 - 32, 72 : 627 - 632) "4567-"
                        @Tarminal : (32, 72 - 32, 76 : 632 - 636) "8904"
        Family : (40, 4 - 43, 0 : 661 - 771) : {}
            FamilyName : (40, 4 - 40, 19 : 661 - 676) : {}
                Literal : (40, 4 - 40, 19 : 661 - 676) : {}
                    @Tarminal : (40, 5 - 40, 17 : 662 - 674) "FamilyName05"
            Person : (41, 16 - 41, 85 : 694 - 763) : {startcolumn: @C >= 11, startline: @L > 40}
                Name : (41, 16 - 41, 36 : 694 - 714) : {}
                    Literal : (41, 23 - 41, 34 : 701 - 712) : {}
                        @Tarminal : (41, 24 - 41, 33 : 702 - 711) "Dddd Aaaa"
                Age : (41, 36 - 41, 46 : 714 - 724) : {}
                    Number : (41, 42 - 41, 44 : 720 - 722) : {}
                        @Tarminal : (41, 42 - 41, 44 : 720 - 722) "43"
                Sex : (41, 46 - 41, 57 : 724 - 735) : {}
                    MorF : (41, 52 - 41, 55 : 730 - 733) : {}
                        Male : (41, 52 - 41, 55 : 730 - 733) : {}
                Phone : (41, 57 - 41, 85 : 735 - 763) : {}
                    NumberHyphen : (41, 65 - 41, 85 : 743 - 763) : {}
                        @Tarminal : (41, 65 - 41, 70 : 743 - 748) "0123-"
                        @Tarminal : (41, 70 - 41, 75 : 748 - 753) "4567-"
                        @Tarminal : (41, 75 - 41, 80 : 753 - 758) "8905-"
                        @Tarminal : (41, 80 - 41, 84 : 758 - 762) "1111"
    _EOF : (43, 2 - 43, 2 : 773 - 773) : {}
